custom-vu/src/main/kotlin/jces1209/vu/TrafficDataParser.kt

```diff
@@ -0,0 +1,24 @@
+package jces1209.vu
+
+import java.io.StringReader
+import javax.json.Json
+import javax.json.JsonReader
+
+class TrafficDataParser {
+
+    companion object {
+
+        fun parseData(hostName: String, readEnvTrafficShapeConfig: String): String {
```

Comment:
not sure if this `readEnvTrafficShapeConfig` is still needed, my opinion is that if the file doesn't exist, just safely fallback to default. Is the class even needed though?